src/index.jade

```diff
@@ -56,29 +56,54 @@ html
 
       .button-demo
         p.lead .btn
-        a.btn Click Me
-        
+        a(tabindex).btn Click Me
```

Comment:
[This line](https://github.com/twbs/bootstrap/blob/2919c3407111ac960c746b78b00ee4f0ae443ad5/dist/css/bootstrap-reboot.css#L331) is messing up the demo, since the buttons aren't links. I don't want to change that style though, as I think it's a good way to point out that we're not using a button properly if it looks broken in our site.